src/js/components/Form/FormContext.js

```diff
@@ -24,5 +24,4 @@ const useFormField = ({ error, info, disabled }) => ({ error, info, disabled });
 export const FormContext = React.createContext({
   useFormField,
   useFormInput,
-  noForm: true,
```

Comment:
I'm a little confused where in the current implementation `noForm` gets set to false

Reply:
Right now, when you wrap something in a Form then it completely overrides this default context value so `noForm` just becomes undefined. And this was only added for the Data + friends scenario, but since we need a bit more specificity that we are in a DataForm, we don't need this generic `noForm` value in FormContext anymore.

Reply:
☝️